src/main/java/by/onliner/page/MainPage.java

```diff
@@ -0,0 +1,40 @@
+package by.onliner.page;
+
+import by.onliner.driver.DriverManager;
+import by.onliner.dto.Menu;
+import by.onliner.util.Waiter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+
+import static by.onliner.constants.OnlinerConstants.*;
+
+
+public class MainPage extends AbstractPage {
+
+    @FindBy (xpath = "//div[@data-index= '1']")
```

Comment:
Try to get rid of indices in the locator or pass them there according to what that index mean

Reply:
Renamed locator to:
//div[contains(@class, 'catalog-offers__item')][1]